talks/de.json

```diff
@@ -1,126 +1,151 @@
 {
 	"respondTimeRemaining": {
 		"default": [
-			"Es sind noch {0} bis zu deinem {1}",
-			"Einen Moment bitte. Du hast noch {0} übrig auf dem {1}",
-			"Insgesamt hast du noch {0} auf dem {1}",
-			"Für den {1} verbleiben noch {0}. Keine Sorge, ich halet dich auf dem laufenden."
+			"Es gibt {0} Zeit links auf dieser {1}",
+			"Bitte halten Sie die Linie, während ich überprüfen. Scheint Sie haben {0} auf die linke {1}",
```

Comment:
`Please hold the line while I check. Looks like You have {0} on the left`

:-)